### PR TITLE
Improvements:  switch account menu, and confirm popup

### LINKED ISSUE #1848: Switch Org -> in profile menu -> No scroll bar if more Org exist for user
1. Open 'https://staging.fightpandemics.work/'
2. Login as user having more than 10 Orgs under their account.
3. click on profile icon -> Switch Account 

Issue: You will observe that user is unable to view/scroll the Organization at the end. there should be scroll if user is having more Orgs.

Refer below screenshot.
image

Another Issue:
The banner is hiding the profile menu.
Refer below screenshot.
image

### LINKED ISSUE #1854: Switch Profile - Add new user selection pop up for user switch profile option
when user click on ‘switch profile’ then pop up should come to ask them if they want to be on same page.., 

"Do you want to be on same page?"
“yes or no.” . 

They will choose yes, if they want to stay in same page where they are like either in Helpboard , about us or any page. else if user choose 'No', then they will be navigated back to their respective Profile page.

### LINKED ISSUE #1857: Switch accounts - login with the default user
env: staging

1. Login with Individual user email 
Screen Shot 2020-11-18 at 21 53 25
2. Switch account to Organization:
Screen Shot 2020-11-18 at 21 55 02
3. Logout
4. Login once more with Individual user email 
Profile is shown for Organization account

### PR DESCRIPTION
**Pull request by Zaydme**
### What does this pull request aim to achieve? 💯
- Fix the profile menu not scrollable when having a lot of orgs.
- Fix the banner (summit announcement) hiding the profile menu.
- Add popup confirmation to stay in the same page or redirect to the help board after switching accounts.
- When the user logs out, clear the selected organization.

_Please be concise and link any related issue(s):_
closes #1848 
closes #1854 
closes #1857

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [x] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

File: client/src/templates/layouts/NavigationLayout.js [hunk 1]
```diff
@@ -67,10 +67,41 @@ const NavigationLayout = (props) => {
...
+            {
+              text: t("common.no"),
+              onPress: () => (window.location.href = "/feed"),
+              style: "default",
+            },
+            { text: t("common.yes"), onPress: () => window.location.reload() },
+          ],
+        );
+      } else if (window.location.href.includes("/feed")) {
+        Modal.alert(null, t("common.willRefresh"), [
```
Comment (ddveloper) [inline client/src/templates/layouts/NavigationLayout.js:96]: I think this pop-up is kind of redundant, since user doesn't have option any way

**Comment by galeza** [comment]
Hi,

I tested the following scenarios:
1. Tested for a large amount of orgs: Drop down is scrollable - PASS
2. Open About us page - switch account - Popup is shown Stay on the current page?. Yes - stays at the same page, No - redirected to the Help Board. - PASS
3. Open Help Board - switch account - Popup is shown Refresh page. Click Ok. Page is refreshed.
@Zaydme would it be possible in the future to press Enter and confirm OK? Or maybe do this automatically? 
I would use OK - both capital letters or Confirm?
4. Banner doesn't cover Profile dropdown - PASS 
5. Create new org - click on the dropdown list and select Add org - PASS

File: client/src/templates/layouts/NavigationLayout.js (hunk 1 above)
Comment (ddveloper) [reply client/src/templates/layouts/NavigationLayout.js:96]: I just understand from @rabbi08 this message is needed to remind user board information will be refreshed

**Comment by galeza** [comment]
@Zaydme 
will this last commit fix Switch accounts - login with the default user #1857

**Review by rabbi08 — APPROVED** [review]
Thanks @Zaydme , validated below scenarios as working fine.

- If user log out as Org and login again -> user logs as individual user acc : PASS
- Profile menu scroll working fine for long list of Org: PASS
- Profile menu is not overlapped by the banner
- User on home page -> Switch Acc -> Pop up  -> yes -> stay on home page -> profile switched in the profile menu 
- User on home page -> Switch Acc -> Pop up  -> No -> navigate to help board page -> profile switched in the profile menu -> post are displayed as per selected user profile
- On post selection page, user can switch multiple accounts and like or comment or share  by staying on same page.
- On help Board page -> switch profile -> Refresh pop up -> ok -> posts are displayed as per the selected user profile location
- On User profile EDIT profile page -> switch acc -> user is redirected to Help Baord and the selected profile is displayed in the profile header
- On help Board -> add filter -> search 'Health' -> select TAB 'Org' -> switch profile -> Refresh pop -> ok Page refresh but search keyword , selected TAB and filters are retained.

**Comment by Zaydme** [comment]
@galeza yes

**Review by ddveloper — APPROVED** [review]
LGTM!

**Pull request merged**